_includes/dochead.html

```diff
@@ -34,7 +34,7 @@
     var trackOutboundLink=function(n){ga("send","event","outbound","click",n,{hitCallback:function(){document.location=n}})};
   </script>
  </head>
- <body>
+ <body class="article">
```

Comment:
Because the API .html files have its own style for <body>, which overrides the style of <body> used by other html. By making the <body> here attached with a class, I can override the style of <body> defined in the two API .html files. Does it make sense?